js/app.js

```diff
@@ -0,0 +1,147 @@
+var object;
+	function show(sum,p){
+		var pp=p.toString;
+		document.getElementById("1").innerHTML="Height";
```

Comment:
you don't have  to call document.getElement every time, call it once and store that in variable, which you can refere later

Reply:
Ok